src/libs/autoScrollback/index.web.js

```diff
@@ -0,0 +1,56 @@
+/* autoScrollBack address Mobile Safari-specific issues when the user overscrolls the window while the keyboard is visible */
+import Str from 'expensify-common/lib/str';
+
+import getBrowser from '../getBrowser';
+
+const init = () => {
+    if (getBrowser() !== 'safari') {
+        return;
+    }
+    const userAgent = navigator.userAgent.toLowerCase();
+
+    if (Str.contains(userAgent, 'iphone os 1')) {
+        const innterHeightWithoutKeyboard = window.innerHeight;
+
+        let waitingForScrollAfterFocus = false;
+
+        let maxScrollY = 0;
+
+        const isIos15 = Str.contains(userAgent, 'iphone os 15_');
+
+        let isTouching = false;
+
+        const scrollback = () => {
+            if (!maxScrollY) {
+                return;
+            }
+            if (!isTouching && window.scrollY > maxScrollY) {
+                window.scrollTo({top: maxScrollY, behavior: 'smooth'});
```

Comment:
Is this Smooth working? You said we need a polyfill, I don't mind using that if this does not work.

Reply:
Kinda... It's a very fast snap back, but not teleporting... Check one of the recordings